### DISCUSSION TITLE
PackageTrackingDetails.CustomerTrackingLink missing

### DISCUSSION THREAD

**Discussion by dave-sewell** [abuzuhri/Amazon-SP-API-CSharp#312] [Q&A] [answered]
The PackageTrackingDetails object returned by calling FulfillmentOutbound.GetPackageTrackingDetails is missing a field corresponding to the customerTrackingLink field described in the API documentation.

Dave

**Comment by dave-sewell** [comment]
The field for currentStatusDescription is also missing, though that is less important.

**Comment by abuzuhri** [comment, marked as answer]
check v1.4.28

**Reply by dave-sewell** [reply]
Thanks for adding those fields.
